app/javascript/react/containers/ReviewFormContainer.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+import ReviewForm from '../components/ReviewForm'
+
+class ReviewFormContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bodyText: "",
+      rating: ""
+    }
+    this.reviewChange = this.reviewChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.ratingChange = this.ratingChange.bind(this)
+  }
+
+  ratingChange(value) {
+    this.setState({ rating: value })
+  }
+
+  handleSubmit() {
+    let formPayload = {
+      review: {
+        body: this.state.bodyText,
+        rating: this.state.rating,
+        user_id: 1,  // this is a temporary hardcorde for testing puroposes, change to: props.userId,
```

Comment:
I wouldn't want to merge this in with the hardcoded value! Can we get this updated yet, or is there something you're waiting on?